e2e/tests/negative/kubelet_restart.robot

```diff
@@ -53,3 +53,46 @@ Stop Volume Node Kubelet For More Than Pod Eviction Timeout While Workload Heavy
         Then Check statefulset 0 works
         And Check statefulset 1 works
     END
+
+Restart Volume Node Kubelet Immediately
+    Given Set setting rwx-volume-fast-failover to ${RWX_VOLUME_FAST_FAILOVER}
+    And Create storageclass longhorn-test with    dataEngine=${DATA_ENGINE}
+    And Create statefulset 0 using RWO volume with longhorn-test storageclass
+    And Create statefulset 1 using RWX volume with longhorn-test storageclass
+    And Write 100 MB data to file data in statefulset 0
+    And Write 100 MB data to file data in statefulset 1
+
+    When Stop volume nodes kubelet for 5 seconds    statefulset 0    statefulset 1
```

Comment:
Should we check the status of volumes and pods after kubelet restart?

Reply:
Updaed for check pod not restart